mgmt/RecordsConfig.cc

```diff
@@ -854,6 +854,8 @@ static const RecordElement RecordsConfig[] =
   //  # how often should the directory be synced (seconds)
   {RECT_CONFIG, "proxy.config.cache.dir.sync_frequency", RECD_INT, "60", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.cache.dir.enable_hugepages", RECD_INT, "0", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
Doesn't look like you added documentation for this.

Also, even though there is not a "bool" type for config variable, I don't think you should propagate the use of an integral type instead of bool for all use of the variable in the code.

Reply:
resolved